### PR TITLE
Update E3SM Diags default sets

### LINKED ISSUE #259: Reconsider which `sets` are default for the `e3sm_diags` task
Reconsider which `sets` are default for the `e3sm_diags` task. Reducing the number of default sets could make it easier for beginners to run `zppy`.

### PR DESCRIPTION
**Pull request by forsyth2**
Update E3SM Diags default sets. Resolves #259.

### CODE REVIEW

File: zppy/templates/default.ini [hunk 1]
```diff
@@ -170,7 +170,8 @@ run_type = string(default="model_vs_obs")
 # "diurnal_cycle" requires `climo_diurnal_subsection`, `climo_diurnal_frequency`, and `dc_obs_climo` to be set.
 # "streamflow" requires `streamflow_obs_ts` to be set.
 # "tc_analysis" requires `tc_obs` to be set.
-sets = string_list(default=list("lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere", "tc_analysis"))
+# All available sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow","zonal_mean_2d_stratosphere","tc_analysis",
+sets = string_list(default=list("lat_lon"))
```
Comment (forsyth2) [inline zppy/templates/default.ini:174]: @chengzhuzhang @tangq I made `lat_lon` the only default set. Is that what we want? If so, I can go ahead and merge this PR.

**Comment by chengzhuzhang** [comment]
Hi @forsyth2 
I think we can keep the core set (meaning that as long as standard climo files are generated, they can run):
"lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d", "annual_cycle_zonal_mean", "zonal_mean_2d_stratosphere"

And change  the comment "All available sets = ..." to "Additional available sets can be included based on availability of input :"diurnal_cycle", "enso_diags","qbo", "streamflow", "tc_analysis". 

For the template suggested in another issue, we can provide more guidance for including additional diags.

**Comment by forsyth2** [comment]
> I think we can keep the core set 

Ok, I updated the default and expanded on the documentation comments.

> For the template suggested in another issue, we can provide more guidance for including additional diags.

Yes, #258 for reference.

**Comment by chengzhuzhang** [comment]
Awesome! Thank you for a quick turn-around!

**Pull request merged**